src/studio/nuxeo-studio/working-in-view-designer/ui-designer.md

```diff
@@ -80,19 +80,39 @@ Finally, when dealing with complex multivalued fields:
 When hovering on an element in a document or workflow task layout, only the trash icon should be used at the moment. Using the sortable toggle (arrows icon) or the visibility toggle (eye icon) may prevent your layout from working properly.
 {{/callout}}
 
+## Layout Blocks
+
+Allows you to create reusable components and share them between different documents layouts.
+Layout blocks are elements of a layout that you can reuse them in other layouts configurations.
+In each layout block, you can insert properties along with their configuration.
+
+Nuxeo Studio provides you a variety of Layout Blocks templates:
+
+- **Title/description element template**: By default, provides an element containing the title and description properties in a single block. Properties can be added to the block.
```

Comment:
This list needs to be reviewed to remove all the repetitions of "Provides an element for"